capa/features/extractors/frida/models.py

```diff
@@ -64,6 +64,9 @@ def from_json_file(cls, json_path) -> "FridaReport":
                 elif "api" in record and "java_api" in record["api"]:
                     call = Call(**record["api"]["java_api"])
                     api_calls.append(call)
+                elif "api" in record and "native_api" in record["api"]:
```

Comment:
Let's me specific here; e.g. `"api" == record["api"]`. Currently, the first check matches both `api` and `native_api` so the second check isn't even necessary. This will make the code easier to maintain morning forward.